assignments/callbacks.js

```diff
@@ -28,24 +30,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
 }
+function getLength(arr,cb){
+  return cb(arr.length);
+}
+getLength(items,function(getLength){
+  console.log(getLength)
+});
```

Comment:
This does exactly what it is supposed to, it would be a little easier to read if `getLength` was named something else in the anonymous function